infra/gcp/main.tf

```diff
@@ -415,17 +415,9 @@ resource "google_artifact_registry_repository_iam_member" "artifact_registry_pus
   member = "serviceAccount:${google_service_account.gcr_push.email}"
 }
 
-# This is intended to match the secret name also used for azure credentials
-# This should ultimately be replaced by using CI's own batch-managed credentials
-# in BuildImage jobs
-resource "kubernetes_secret" "registry_push_credentials" {
-  metadata {
-    name = "registry-push-credentials"
-  }
-
-  data = {
-    "credentials.json" = base64decode(google_service_account_key.gcr_push_key.private_key)
-  }
+module "ukbb" {
```

Comment:
Removing `resource "kubernetes_secret" "registry_push_credentials" `
Diff output:
```
+<<<<<<< HEAD
+# This is intended to match the secret name also used for azure credentials
+# This should ultimately be replaced by using CI's own batch-managed credentials
+# in BuildImage jobs
+resource "kubernetes_secret" "registry_push_credentials" {
+  metadata {
+    name = "registry-push-credentials"
+  }
+
+  data = {
+    "credentials.json" = base64decode(google_service_account_key.gcr_push_key.private_key)
+  }
+}
+
+||||||| dc7fce0e3
+# This is intended to match the secret name also used for azure credentials
+# This should ultimately be replaced by using CI's own batch-managed credentials
+# in BuildImage jobs
+resource "kubernetes_secret" "registry_push_credentials" {
+  metadata {
+    name = "registry-push-credentials"
+  }
+
+  data = {
+    "credentials.json" = base64decode(google_service_account_key.gcr_push_key.private_key)
+  }
+}
+
+module "ukbb" {
+  count = var.deploy_ukbb ? 1 : 0
+  source = "../k8s/ukbb"
+}
+
+=======
 module "ukbb" {
   count = var.deploy_ukbb ? 1 : 0
   source = "../k8s/ukbb"
 }
 
+>>>>>>> 13de4e6ff
```

Reply:
(One of) our local-branch changes in this file is to remove the `module "ukbb"` clause. Their recent change is to remove the adjacent `resource "kubernetes_secret"` clause.

So in our merge, we should remove **both** of these clauses.

Reply:
module ukbb removed